static-site-plugin.js

```diff
@@ -26,10 +27,11 @@ module.exports = class {
       const { filename } = this
       const { path: outputPath } = outputOptions
       const { options } = compiler
-      const { assets: clientAssets, outputOptions: clientOutput } = options.clientCompilation || {}
-      const appAssets = Object.keys(clientAssets).map((asset) => (
-        clientOutput.publicPath + asset
-      ))
+      const { clientStats = {} } = options
+      const { compilation: clientCompilation = { toJson () { return {} } } } = clientStats
```

Comment:
You're right. A better option would be a descriptive error message when trying to use static render with a web pack config which doesn't have a `clientStats` key...

I'm also sort of wondering if it isn't backwards... that perhaps the client compilation should have the static render plugin attached, and should be passed a server-app instance. Thinking about this stuff has me a little more convinced that the `static-site-plugin` could be internal to brb. Something that can be turned on with a flag. Anyway. For another PR